eng/pipelines/templates/jobs/archetype-sdk-nightly.yml

```diff
@@ -100,32 +105,3 @@ jobs:
               artifactName: 'artifacts' 
               targetPath: $(Build.ArtifactStagingDirectory)
         ToxTestEnv: "whl,sdist"
-
-  - job: 'Test_Linux_Python38'
-    variables:
-    - template: ../variables/globals.yml
-
-    dependsOn:
-       - 'Build'
-
-    timeoutInMinutes: 300
-
-    pool:
-      vmImage: 'ubuntu-16.04'
-
-    steps:
-    - template: ../steps/test-nightly-python38.yml
```

Comment:
Should we delete `../steps/test-nightly-python38.yml` as well?

Reply:
yes. I thought of submitting separate PR to delete them once this change is successfully merged. Please let me know if you want me to include that change in this PR itself.